packages/react-native-sdk/src/hooks/useScreenShareButton.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  getLogger,
+  hasScreenShare,
+  OwnCapability,
+} from '@stream-io/video-client';
+import { useCall, useCallStateHooks } from '@stream-io/video-react-bindings';
+import React, { useRef, useEffect } from 'react';
+import { Platform, findNodeHandle, NativeModules } from 'react-native';
+import { usePrevious } from '../utils/hooks';
+import { useIsIosScreenshareBroadcastStarted } from './useIsIosScreenshareBroadcastStarted';
+
+// ios >= 14.0 or android - platform restrictions
+const CanDeviceScreenShare =
+  (Platform.OS === 'ios' &&
+    // @ts-ignore
+    Number.parseInt(Platform.Version.split('.')[0], 10) >= 14) ||
```

Comment:
```suggestion
    Number.parseInt(Platform.Version.split('.')[0], 10) >= 14, 10) ||
```

Please always make sure to pass `radix` to the `parseInt` function as otherwise weird things can happen:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#radix

Reply:
thanks